apps/spruce/src/pages/waterfall/useFilters.ts

```diff
@@ -0,0 +1,98 @@
+import { useMemo } from "react";
+import { WaterfallQuery, WaterfallVersionFragment } from "gql/generated/types";
+import { useQueryParam } from "hooks/useQueryParam";
+import { WaterfallFilterOptions } from "types/waterfall";
+
+export const useFilters = (waterfall: WaterfallQuery["waterfall"]) => {
```

Comment:
should the filters clear when switching to a different project? (not sure if this is captured by another ticket 😅)